src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c

```diff
@@ -1009,12 +1032,10 @@ static inline int process_ert_rq(struct xocl_ert_user *ert_user)
 
 		//sched_debug_packet(epkt, epkt->count+sizeof(epkt->header)/sizeof(u32));
 
-		if (cmd_opcode(ecmd) == OP_CONFIG && !ert_user->polling_mode) {
-			for (i = 0; i < ert_user->num_slots; i++) {
-				xocl_intc_ert_request(xdev, i, ert_user_isr, ert_user);
-				xocl_intc_ert_config(xdev, i, true);
-			}
-
+		if (cmd_opcode(ecmd) == OP_CONFIG) {
+			ert_user->is_configured = false;
+			if (!ert_user->polling_mode)
+				ert_intc_enable(ert_user, true);
```

Comment:
intr is disabled in _remove routine, why intr is not enabled in _probe()?

Reply:
We enable interrupt and register related callback function if polling_mode = true. This flag is as part of OP_CONFIG command which  is given by xrt.ini for now and kds in the future. 

So we should disable and unregister in the same path.

And we also have to clean it up in "ert_user_exit" function